src/components/Editor/Invitees/BbbAddRoomActions.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+	<NcActions v-if="isCreateBbbRoomButtonVisible"
+		:menu-title="$t('calendar', 'Add video link')"
```

Comment:
**suggestion:** I have the feeling it would be best to mention BBB in the label.